pkg/apply/apply.go

```diff
@@ -14,9 +14,9 @@ import (
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// ApplyObject applies the desired object against the apiserver,
+// Object applies the desired object against the apiserver,
 // merging it with any existing objects if already present.
-func ApplyObject(ctx context.Context, client k8sclient.Client, obj *uns.Unstructured) error {
+func Object(ctx context.Context, client k8sclient.Client, obj *uns.Unstructured) error {
```

Comment:
this is not a descriptive name for this funcion. ApplyObject is more meaningful

Reply:
Also, this is unrelated to the travis setup

Reply:
$ golint
apply.go:19:6: func name will be used as apply.ApplyObject by other packages, and that stutters; consider calling this Object